cime/src/drivers/mct/main/cime_comp_mod.F90

```diff
@@ -1887,7 +1887,7 @@ subroutine cime_init()
     !----------------------------------------------------------
 
     areafact_samegrid = .false.
-#if (defined BFB_CAM_SCAM_IOP )
+#if (defined E3SM_SCM_REPLAY )
     if (.not.samegrid_alo) then
        call shr_sys_abort(subname//' ERROR: samegrid_alo is false - Must run with same atm/ocn/lnd grids when configured for scam iop')
```

Comment:
Not essential but just to have a more consistent message. change "when confihgured for scam iop" to " ... for scm replay". Similarly for line 1702.